test/unit/krane/kubernetes_resource/pod_test.rb

```diff
@@ -220,7 +220,7 @@ def test_deploy_failed_is_true_for_disappeared_unmanaged_pods
     template = build_pod_template
     pod = Krane::Pod.new(namespace: 'test', context: 'nope', definition: template,
       logger: @logger, deploy_started_at: Time.now.utc)
-    cache = build_resource_cache
+    cache = build_resource_cache(global_kinds: nil)
```

Comment:
`build_resource_cache` stubs `@task_config.global_kinds` (which gets called by `fetch_by_kind`). Because we've configured the fail on unused stubs we need an opt-out mechanism. Want me to change the name? The other option is to explicitly opt-in, but seemed odd since its the default behavior.

Reply:
Oh, I see. My suggestion would be to make the signature `def build_resource_cache(task_config: build_task_config)`, allowing you to pass in a task config with different stubbing if you need to. Looking at this again, it is a little odd to be passing the builder `global_kinds`, which is not a property of `ResourceCache`.